.travis.yml

```diff
@@ -85,6 +85,7 @@ script:
       else
         py.test -m "not avoid_travis" distributed $PYTEST_OPTIONS;
       fi;
+    - flake8 distributed/*.py distributed/{bokeh,protocol,deploy}/*.py # no tests yet
```

Comment:
Yes. You can just specify `flake8 --exclude tests distributed`. Can also add this to the `setup.cfg` as:

```
[flake8]
exclude = tests
```